### DISCUSSION TITLE
Schedule task to kubernetes Pod

### DISCUSSION THREAD

**Discussion by teaglebuilt** [apache/airflow#15988] [Q&A]
So I do not want to create a new pod for every task..I have a factory approach that dynamically creates the tasks / operators would like some if not all of these tasks to be executed on the same pod.

Is there any use cases out there for this? Examples?

Trying to determine if I need to use a custom executor for this or inherit the KubernetesPodOperator into where my BaseOperator class is when the operator is instantiated.

the reattach_on_restart field on the kubernetes pod operator seems to offer scheduling the task to the pod if the scheduler fails....so can we just do this regardless?

Does anyone have a suggestion on which path to take?

**Comment by viswanathareddy04** [comment]
@teaglebuilt I'm looking for the same feature. I want to make use of the same pod for the next scheduled run. Did you find any luck?
